.github/ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,32 @@
+### Version of Singularity:
 
-## Version of Singularity:
+Write your Singularity version here.
 
-Write here.
 
 ### Expected behavior
 
-Write here.
+What did you expect to see when you do...?
+
 
 ### Actual behavior
 
-Write here.
+What actually happend? Why was it incorrect?
+
 
 ### Steps to reproduce behavior
 
-Write here.
+Please fill out as much information as possible.
+
+#### What OS are you running
```

Comment:
I suggest to have a single environment section like [here](https://github.com/sylabs/singularity-cri/blob/master/.github/ISSUE_TEMPLATE/bug_report.md#environment).

And remove `Version of Singularity`, `What OS are you running` and `How did you install Singularity`.

Reply:
I added a `Environment` section like suggested, but I think we should keep the `Version of Singularity` section. Other sections can be added/removed later if we find out that they are not useful.